bulletin_board/server/cypress/integration/sandbox/election.page.js

```diff
@@ -109,15 +109,15 @@ export class ElectionPage {
               },
               {
                 force: true,
-              }
+              },
             );
 
           cy.findByText("Start").click();
 
           cy.findByText(name)
             .parent("tr")
             .within(() => {
-              cy.findByText("Backup").click({ timeout: 180_000 });
+              cy.findByText("Backup").click({ timeout: 720_000 });
```

Comment:
Is it really necessary quadrupling this timeout or maybe just duplicating could be enough? 
```suggestion
              cy.findByText("Backup").click({ timeout: 360_000 });
```

Reply:
No, it's not. Already tested (just look at the previous commit)

Reply:
I think you mean to say "Yes, it is" 😄 
I've missed that commit, it makes sense!!